webapp/channels/src/components/dropdown_input.tsx

```diff
@@ -169,6 +170,7 @@ const DropdownInput = <T extends ValueType>(props: Props<T>) => {
                         value={value}
                         onChange={ownOnChange as any} // types are not working correctly for multiselect
                         styles={{...baseStyles, ...styles}}
+                        getOptionLabel={(option) => getOptionLabel(option, intl)}
```

Comment:
I don't know if this is actually needed here since `ValueType` has `label` as a string